### PR TITLE
Replaced all the images with SVG SFSymbols

### LINKED ISSUE #89: Replace icons with SFSymbols
Use `SFSymbols` only for `iOS 13`.

### PR DESCRIPTION
**Pull request by serjooo**
resolves #89

### CODE REVIEW

**Comment by jdisho** [comment]
This is great! Thank you! @serjooo 

I will review this PR asap 😊

File: Papr/Commons/Constants.swift [hunk 1]
```diff
@@ -117,16 +117,30 @@ enum Constants {
...
+            static var doneWhiteSmall: UIImage {
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "checkmark", withConfiguration: symbolConfigurationSmall)!.withTintColor(.white)
+                }
+                return UIImage(imageLiteralResourceName: "done")
+            }
+
+            static var addSmall: UIImage {
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "plus", withConfiguration: symbolConfigurationSmall)!.withTintColor(.white)
```
Comment (jdisho) [inline Papr/Commons/Constants.swift]: I think `addSmall` image should have a **medium** configuration since it is being used in a `UIBarButtonItem`.

**Review by jdisho — CHANGES_REQUESTED** [review]
LGTM 🎉 

However, can you also remove the references from the .xib files? I see these warnings in console: 

```
2019-10-13 16:01:28.539046+0200 Papr[7270:328475] Could not load the "eye" image referenced from a nib in the bundle with identifier "disho.Papr"
2019-10-13 16:01:28.540577+0200 Papr[7270:328475] Could not load the "favorite-border-white" image referenced from a nib in the bundle with identifier "disho.Papr"
```

**Comment by serjooo** [comment]
@jdisho changes are done 👍🏻

File: Papr/Commons/Constants.swift [hunk 2]
```diff
@@ -117,16 +117,30 @@ enum Constants {
...
+            }
+
+            static var doneWhiteSmall: UIImage {
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "checkmark", withConfiguration: symbolConfigurationSmall)!.withTintColor(.white)
+                }
+                return UIImage(imageLiteralResourceName: "done")
+            }
+
+            static var addSmall: UIImage {
```
Comment (jdisho) [inline Papr/Commons/Constants.swift]: nit: Can we rename this to `addMedium`?  (b/c it is using a medium configuration)

File: Papr/Commons/Constants.swift [hunk 3]
```diff
@@ -117,16 +117,30 @@ enum Constants {
...
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "plus", withConfiguration: symbolConfigurationMedium)!.withTintColor(.white)
```
Comment (jdisho) [inline Papr/Commons/Constants.swift]: Can we remove tinting? Otherwise theming wouldn't work on this. 🤔

**Review by jdisho — CHANGES_REQUESTED** [review]

File: Papr/Commons/Constants.swift (hunk 2 above)
Comment (serjooo) [reply Papr/Commons/Constants.swift]: Completely my bad! Will do this tonight!

File: Papr/Commons/Constants.swift (hunk 3 above)
Comment (serjooo) [reply Papr/Commons/Constants.swift]: It was previously `addSmallWhite` so I wasn't sure if I should hardcode it or not. Will remove it as well 👍🏻

**Review by jdisho — APPROVED** [review]
🚀

**Pull request merged**